portality/static/doaj/js/available_facetviews/reapplications.js

```diff
@@ -1,203 +1,242 @@
-        {'field': 'bibjson.provider.exact', 'display': 'Provider'},
-        {'field': 'bibjson.author_pays.exact', 'display': 'Publication charges?'},
-        {'field': 'index.license.exact', 'display': 'Journal License'},
-        {'field': 'bibjson.oa_start.exact', 'display': 'Started publishing OA content (year)'},
-        {'field': 'bibjson.oa_end.exact', 'display': 'Stopped publishing OA content (year)'},
-    ],
-    search_sortby: [
-        {'display':'Date added to DOAJ','field':'created_date'},
-        {'display':'Last updated','field':'last_updated'},
-        {'display':'Title','field':'index.title.exact'},
-    ],
-    searchbox_fieldselect: [
-        {'display':'Title','field':'index.title'},
-        {'display':'Journal: Alternative Title','field':'bibjson.alternative_title'},
-        {'display':'Subject','field':'index.subject'},
-        {'display':'Classification','field':'index.classification'},
-        {'display':'ISSN', 'field':'index.issn.exact'},
-        {'display':'Journal Country','field':'index.country'},
-        {'display':'Journal Language','field':'index.language'},
-        {'display':'Publisher','field':'index.publisher'},
-        {'display':'Journal: Provider','field':'bibjson.provider'},
-    ],
-    paging: {
-      from: 0,
-      size: 10
-    },
-    default_operator: "AND",
-    result_display: [
-        [
-            {
-                "field" : "title_field"
-            }
-        ],
-        [
-            {
-                "pre": '<span class="alt_title">Alternative title: ',
-                "field": "bibjson.alternative_title",
-                "post": "</span>"
-            }
-        ],
-        [
-            {
-                "pre": "<strong>Date applied</strong>: ",
-                "field": "suggested_on",
-            }
-        ],
-        [
-            {
-                "pre": "<strong>Last updated</strong>: ",
-                "field": "last_updated",
-            }
-        ],
-        [
-            {
-                "pre" : "<strong>ISSN(s)</strong>: ",
-                "field" : "issns"
-            }
-        ],
-        [
-            {
-                "pre" : "<strong>Application status</strong>: ",
-                "field" : "admin.application_status"
-            }
-        ],
-        [
-            {
-                "pre" : "<strong>Description</strong>: ",
-                "field" : "suggestion.description"
-            }
-        ],
-        [
-            {
-                "pre" : "<strong>Contact</strong>: ",
-                "field" : "admin.contact.name"
+jQuery(document).ready(function ($) {
+
+    var status_mapping = {
+        "reapplication" : {"text" : "pending" }
+    };
+
+    function doCustomise() {
+        // do the standard results customisation
+        customise_facetview_results();
+
+        // application status
+        // 1. facet itself
+        $('#facetview_admin_application_status_exact .facetview_filterchoice_text').each(
```

Comment:
Yes, that made the comparison confusing, but the indenting in the file was
driving me nuts :)  That is the addition to this file, yeah.

On 7 January 2015 at 10:03, Emanuil Tolev <notifications@github.com> wrote:

> In portality/static/doaj/js/available_facetviews/reapplications.js
> <https://github.com/DOAJ/doaj/pull/503#discussion-diff-22578692>:
>
> > -            {
> > -                "pre" : "<strong>Contact</strong>: ",
> > -                "field" : "admin.contact.name"
> > +jQuery(document).ready(function ($) {
> > +
> > +    var status_mapping = {
> > +        "reapplication" : {"text" : "pending" }
> > +    };
> > +
> > +    function doCustomise() {
> > +        // do the standard results customisation
> > +        customise_facetview_results();
> > +
> > +        // application status
> > +        // 1. facet itself
> > +        $('#facetview_admin_application_status_exact .facetview_filterchoice_text').each(
>
> File's been reformatted but this looks like the new code. Looks ok!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DOAJ/doaj/pull/503/files#r22578692>.
>



-- 

Richard Jones,

Founder, Cottage Labs
t: @richard_d_jones, @cottagelabs
w: http://cottagelabs.com